clients/ts/FunctionalTests/Program.cs

```diff
@@ -3,7 +3,9 @@
 
 using System;
 using System.IO;
+using System.Security.Cryptography.X509Certificates;
 using Microsoft.AspNetCore.Hosting;
+using Microsoft.Extensions.Configuration;
```

Comment:
Nit: The ConfigurationBuilder should no longer be needed now that config.json is removed.